WooCommerce/Classes/ViewRelated/Products/ProductDetailsViewController.swift

```diff
@@ -0,0 +1,388 @@
+import UIKit
+import Yosemite
+
+
+/// ProductDetailsViewController: Displays the details for a given Product.
+///
+class ProductDetailsViewController: UIViewController {
```

Comment:
A bit of a nitpick but would you consider declaring this class as final?

Reply:
Indeed!